addons/gamification_sale_crm/data/gamification_sale_crm_data.xml

```diff
@@ -102,8 +102,8 @@
             <field name="domain">[('state','!=','cancel'),('user_id','=',user.id),('type','=','out_refund')]</field>
         </record>
         <record model="gamification.goal.definition" id="definition_crm_tot_customer_refunds">
-            <field name="name">Total Customer Refunds</field>
-            <field name="description">The total refunded value is a negative value. Validated when higher (min refunded).</field>
+            <field name="name">Total Customer Credit Notes</field>
+            <field name="description">The value of total credit note is a negative. Validated when higher (min credit note value).</field>
```

Comment:
The total credit note value is negative. Validated when higher (min credit note value).